src/containers/ModalTweetReplyForm.tsx

```diff
@@ -0,0 +1,15 @@
+import { Chat } from "react-bootstrap-icons";
+
+import TweetReplyForm from "../components/TweetReplyForm";
+import ModalUniversal from "./ModalUniversal";
+
+const ModalTweetReplyForm: React.FC = () => {
+  const TweetRepForm = <TweetReplyForm />;
```

Comment:
propose delete TweetRepForm constant and add <TweetReplyForm /> to <ModalUniversal .../> as '{<TweetReplyForm />}'